debian/rules

```diff
@@ -1,18 +1,38 @@
 #!/usr/bin/make -f
+#
+# Build Debian package using https://github.com/spotify/dh-virtualenv
+#
 
-# This file was automatically generated by stdeb 0.8.2 at
-# Fri, 12 Jun 2015 14:32:03 +0100
-export PYBUILD_NAME=matrix-synapse
-%:
-	dh $@ --with python2 --with systemd --buildsystem=pybuild --no-guessing-deps
+export DH_VIRTUALENV_INSTALL_ROOT=/opt/venvs
+export CFLAGS=-march=x86-64
+SNAKE=/usr/bin/python3
+DH_VENV_ARGS=--install-suffix "matrix-synapse" \
+			 --builtin-venv\
```

Comment:
could you add a comment somewhere here that explains a little bit about why we use builtin-venv?

Reply:
done